clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/ops/CreateAzureServerGroupAtomicOperation.groovy

```diff
@@ -97,17 +95,16 @@ class CreateAzureServerGroupAtomicOperation implements AtomicOperation<Map> {
       description.appName = description.application
 
       // get the app gateway. Verify that it can be used for this server group/cluster. add an app pool if it doesn't already exist
-      def appGateway = description.credentials.networkClient.getAppGateway(resourceGroupName, description.loadBalancerName)
-      if (appGateway) {
-        if (appGateway.tags?.cluster && appGateway.tags?.cluster != description.clusterName) {
-          throw new RuntimeException("Selected Load Balancer: $description.loadBalancerName is already in use by another cluster: $appGateway.tags.cluster")
-        }
-      }
-      else {
-        throw new RuntimeException("Selected Load Balancer $description.loadBalancerName does not exist")
+      // TODO: replace appGatewayName with loadBalancerName
+      if (!description.appGatewayName) description.appGatewayName = description.loadBalancerName
```

Comment:
The formatting here makes this a little tough to read. Please surround the assignment with braces and move the block to its own line.